src/main/java/club/sk1er/patcher/screen/render/overlay/ImagePreview.java

```diff
@@ -148,11 +154,38 @@ private void loadUrl(String url) {
             connection = (HttpURLConnection) u.openConnection();
             connection.setRequestMethod("GET");
             connection.setUseCaches(true);
+            connection.setInstanceFollowRedirects(true);
             connection.addRequestProperty("User-Agent", "Patcher Image Previewer");
             connection.setReadTimeout(15000);
             connection.setConnectTimeout(15000);
             connection.setDoOutput(true);
 
+            if (connection.getHeaderField("Content-Type").contains("text/html")) {
```

Comment:
Should be equals.

Reply:
Equals won't work if the header contains encoding

Reply:
Possibly replace with startsWith?